packages/web/src/pages/pay-and-earn-page/PayAndEarnPage.tsx

```diff
@@ -23,8 +54,70 @@ export const PayAndEarnPage = () => {
   }
   const status = combineStatuses(statuses)
 
+  const {
+    count: salesCount,
+    data: sales,
+    fetchMore: fetchMoreSales,
+    onSort: onSalesSort,
+    onClickRow: onSalesClickRow,
+    isEmpty: isSalesEmpty,
+    isLoading: isSalesLoading,
+    downloadCSV: downloadSalesCSV
+  } = useSales()
+  const {
+    count: purchasesCount,
+    data: purchases,
+    fetchMore: fetchMorePurchases,
+    onSort: onPurchasesSort,
+    onClickRow: onPurchasesClickRow,
+    isEmpty: isPurchasesEmpty,
+    isLoading: isPurchasesLoading,
+    downloadCSV: downloadPurchasesCSV
+  } = usePurchases()
+  const {
+    count: withdrawalsCount,
+    data: withdrawals,
+    fetchMore: fetchMoreWithdrawals,
+    onSort: onWithdrawalsSort,
+    onClickRow: onWithdrawalsClickRow,
+    isEmpty: isWithdrawalsEmpty,
+    isLoading: isWithdrawalsLoading,
+    downloadCSV: downloadWithdrawalsCSV
+  } = useWithdrawals()
+
   const header = <Header primary={messages.title} />
 
+  const tables: Record<TableType, TableMetadata> = useMemo(
```

Comment:
i don't think this needs to be memoized

Reply:
can you tell me why? just small enough that it doesn't matter?

Reply:
It doesn't look like it would be computationally expensive. Is it a bottleneck/slow? Otherwise might be a premature optimization.

https://maxrozen.com/understanding-when-use-usememo